docs/README.md

```diff
@@ -0,0 +1,4 @@
+# Data
```

Comment:
Is this file supposed to live under `data/`?

Reply:
Yes, currently there is no README.me there, so there is no explanation as to how the scripts there are used.

Reply:
Sorry, I meant "Should this file be under the `/docs` folder, or under the `/data` folder?"

But I'm operating under the assumption it is where it should be, so approving.